lib/ansible/modules/network/dcnm/dcnm_vrf.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import json
+from ansible.module_utils.connection import Connection
+from ansible.module_utils.basic import AnsibleModule
+
+__copyright__ = "Copyright (c) 2019 Cisco and/or its affiliates."
+__license__ = "Cisco Sample Code License, Version 1.0"
+__author__ = "Shrishail Kariyappanavar"
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: dcnm_vrf
+short_description: Send REST API requests to DCNM controller for VRF operations
+version_added: "2.10"
+description:
+    - "Send REST API requests to DCNM controller for VRF operations - Create, Attach, Deploy"
+options:
+  action:
+    description:
+    - 'Intended VRF operation - Create, Attach or Deploy'
+    choices: ['Create', 'Attach', 'Deploy']
+    required: yes
+    type: str
+  fabric:
+    description:
+    - 'Name of the target fabric for VRF operations'
+    type: str
+    required: yes
+  vrfName:
```

Comment:
I *think* that ansible has a style enforcement that will force us to change from camelCase to snake_case.

Reply:
Ok, will change that. Thanks.